pages/_app.js

```diff
@@ -1,8 +1,16 @@
 import GlobalStyle from "../styles";
+import { createGlobalStyle } from "styled-components";
+
+const TheLook = createGlobalStyle`
+  body {
+    background-color: #145A32;
+  }
+`;
```

Comment:
You should better define your global style (body, background-color) in the GlobalStyle-component.